tools/NuGet.exe.readme.txt

```diff
@@ -0,0 +1,7 @@
+This is the custom version of NuGet used by OctoPack.
+
+The primary customization is to preserve version strings as "what you see is what you get" for both the filename and metadata.
+
+This is a fork of NuGet 3 available here: https://github.com/OctopusDeploy/NuGet.Client
+The build is available here: http://build.octopushq.com/project.html?projectId=OctopusDeploy_NuGet&tab=projectOverview
```

Comment:
I put it there for convenience, plus it's restricted to only people on our domain.